### PR TITLE
chore: add quotes

### PR DESCRIPTION
**Pull request by donmckenna**
Added quotes from DJ and Manus ❤️ 

Screen Shot 2023-02-14 at 1 38 25 PM

### CODE REVIEW

File: docs/data.ts [hunk 1]
```diff
@@ -269,6 +247,28 @@ export const testimonialCardData: TestimonialCardData[] = [
...
+  },
+  {
+    author: {
+      name: 'Manus Nijhoff',
+      title: 'Co-founder at <a href="https://touchystudios.com/" target="_blank" rel="nofollow">Touchy Studios</a> & full-stack developer at <a href="https://100k.studio/" target="_blank" rel="nofollow">100k</a>',
+    },
+    imgPath: {
+      author: '/img/testimonials/manus_nijhoff.png',
+    },
+    quote: `Running chess in a terminal, running a terminal in the browser, check mate!<br/><strong>The best position to be in is a creative one and the Stackblitz WebContainers allow that.</strong>`,
```
Comment (ykmsd) [inline docs/data.ts:270]:
```suggestion
    quote: `Running chess in a terminal, running a terminal in the browser, check mate!<br/><strong>The best position to be in is a creative one and the StackBlitz WebContainers allow that.</strong>`,
```

Comment (ykmsd) [reply docs/data.ts:270]: Are we allowed to modify the quote? 😬

File: docs/data.ts [hunk 2]
```diff
@@ -269,6 +247,28 @@ export const testimonialCardData: TestimonialCardData[] = [
...
+    author: {
+      name: 'DJ',
+      title: 'Founder & CEO @ <a href="https://retune.so/" target="_blank" rel="nofollow">re:tune</a>',
+    },
+    imgPath: {
+      author: '/img/testimonials/retune-dj.jpg',
+      logoDark: '/img/testimonials/retune.png',
+      logoLight: '/img/testimonials/retune.png',
+    },
+    quote: `At re:tune, we have been building the missing frontend for GPT-3, on a mission to empower everyone to build AI-first software at the speed of thought. <strong>WebContainer sets the stage for our AI-native IDE</strong> - with a copilot that can not only read and write code, but can also understand and operate in the full runtime context across server and client!`,
```
Comment (AleksandrSl) [inline docs/data.ts:260]: Should it be `WebContainers`?

File: docs/data.ts (hunk 1 above)
Comment (sylwiavargas) [reply docs/data.ts:270]: yes.

File: docs/data.ts (hunk 2 above)
Comment (sylwiavargas) [reply docs/data.ts:260]: yes

Comment (donmckenna) [reply docs/data.ts:260]: Perhaps.. though it was the direct quote 🤷 
@sylwiavargas ?

Comment (sylwiavargas) [reply docs/data.ts:260]:
```suggestion
    quote: `At re:tune, we have been building the missing frontend for GPT-3, on a mission to empower everyone to build AI-first software at the speed of thought. <strong>WebContainers set the stage for our AI-native IDE</strong> - with a copilot that can not only read and write code, but can also understand and operate in the full runtime context across server and client!`,
```

Comment (sylwiavargas) [reply docs/data.ts]: That's fine, it's a minor change that doesn't impact the meaning 💙 We'll also reach out to the quoted folks.

**Review by ykmsd — APPROVED** [review]
❤️🚀

**Pull request merged**